SofaGLFW/src/SofaGLFW/SofaGLFWBaseGUI.cpp

```diff
@@ -489,23 +491,66 @@ void SofaGLFWBaseGUI::error_callback(int error, const char* description)
     SOFA_UNUSED(error);
     msg_error("SofaGLFWBaseGUI") << "Error: " << description << ".";
 }
- 
+
+int SofaGLFWBaseGUI::handleArrowKeys(int key)
+{
+    // Handling arrow keys with custom codes
+    switch (key)
+    {
+            case GLFW_KEY_UP: return 19;   // Custom code for up
+            case GLFW_KEY_DOWN: return 21; // Custom code for down
+            case GLFW_KEY_LEFT: return 18; // Custom code for left
+            case GLFW_KEY_RIGHT: return 20; // Custom code for right
+    }
+    // Default case return the given value as GLFW handle it
+    return key;
```

Comment:
are you sure there are only 4 exceptions? Have you checked that the other keys work as expected?

Reply:
I have checked that Alphabet Keys ,Number keys , special character keys and also arrow keys work using the keyEvents.py file located examples